docs/administrationGuide.md

```diff
@@ -189,12 +194,12 @@ This library contains a simple command line Lightweight M2M Client that can be u
 on how to perform these kind of tests, see the How-To's in the [Getting Started section of the User Manual](userGuide.md#gettingstarted)
 that shows simple registrations and send measure tests.
 
-# <a name="diagnosis"> Diagnosis procedures </a>
+# Diagnosis procedures
```

Comment:
Now index link to this subsection does not work

Reply:
Fixed - c2bb99a . ToC updated